selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/taskManagement/TaskManagementLocator.java

```diff
@@ -0,0 +1,155 @@
+package org.exoplatform.platform.qa.ui.selenium.locator.taskManagement;
```

Comment:
Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings